homeassistant/components/notify/netgear_lte.py

```diff
@@ -0,0 +1,45 @@
+"""Netgear LTE platform for notify component.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/notify.netgear_lte/
+"""
+
+import voluptuous as vol
+import attr
+
+from homeassistant.components.notify import (
+    BaseNotificationService, ATTR_TARGET, PLATFORM_SCHEMA)
+from homeassistant.const import CONF_HOST
+import homeassistant.helpers.config_validation as cv
+
+from ..netgear_lte import DATA_KEY
+
+
+DEPENDENCIES = ['netgear_lte']
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Optional(CONF_HOST): cv.string,
+    vol.Optional(ATTR_TARGET): vol.All(cv.ensure_list, [cv.string]),
```

Comment:
Without a target here and with no target in the service data, the service won't send the message. I think that is counterintuitive. I suggest making target here required.

Reply:
That seems odd for someone that does not want a default target. I don't really care either way, though, so I have implemented your suggestion.

Reply:
We should at least warn the user why no message was sent if we keep target optional.